openmc_plotter/plotmodel.py

```diff
@@ -742,14 +700,21 @@ def _do_op(array, tally_value, ax=0):
         data_min = np.min(data)
         data_max = np.max(data)
 
-        # set image data, reverse y-axis
-        image_data = data[::-1, ...]
+        # Get mesh bins from openmc.lib
+        mesh_cpp = openmc.lib.meshes[mesh.id]
+        mesh_bins = mesh_cpp.get_plot_bins(
```

Comment:
Before passing the origin to this method, we'll want to account for a mesh translation if one is applied on the `MeshFilter`.